src/jit/compiler.hpp

```diff
@@ -4280,6 +4280,42 @@ bool                Compiler::compIsForInlining()
     return (impInlineInfo != NULL);
 }
 
+//------------------------------------------------------------------------------
+// compIsUnixAmd64ABI : Returns true if the compiler is for compiling UnixAmd64 ABI.
+//
+// Arguments:
+//    None
+//
+// Return Value:
+//    True if the compiler is for compiling UnixAmd64ABI. False otherwise.
+inline
+bool Compiler::compIsUnixAmd64ABI()
+{
+#ifdef UNIX_AMD64_ABI
+    return true;
+#else // !UNIX_AMD64_ABI
+    return false;
+#endif // UNIX_AMD64_ABI
+}
+
+//------------------------------------------------------------------------------
+// compIsAmd64ABI : Returns true if the compiler is for compiling Amd64 ABI.
+//
+// Arguments:
+//    None
+//
+// Return Value:
+//    True if the compiler is for compiling Amd64ABI. False otherwise.
+inline
+bool Compiler::compIsAmd64ABI()
```

Comment:
As Carol says we probably don't need this function as it just means Amd64 TARGET.

Instead we can probably put all of the Windows ABI changes under isWindowsAmd64ABI()